web_src/less/markup/content.less

```diff
@@ -477,12 +477,37 @@
     line-height: 10px;
     color: var(--color-text-light);
     vertical-align: middle;
-    background-color: var(--color-markdown-code-block);
+    background-color: var(--color-markup-code-block);
     border: 1px solid var(--color-secondary);
     border-radius: 3px;
     box-shadow: inset 0 -1px 0 var(--color-secondary);
   }
 
+  .csv-data td,
+  .csv-data th {
+    padding: 5px;
+    overflow: hidden;
+    font-size: 12px;
+    line-height: 1;
+    text-align: left;
+    white-space: nowrap;
+  }
+
+  .csv-data .blob-num {
+    padding: 10px 8px 9px;
+    text-align: right;
+    border: 0;
+  }
+
+  .csv-data tr {
+    border-top: 0;
+  }
+
+  .csv-data th {
+    font-weight: 600;
+    border-top: 0;
+  }
+
```

Comment:
Where do these come from? I see no related CSS class changes, are they dead code?

Reply:
no idear either

Reply:
I think @HarvsG might have added them during testing and Github's CSS includes a same-name class but given that we never actually output such HTML in first place, let's remove it. If  CSV rendering it added to our markdown, the CSS for it should be added along with that.

Reply:
Removed them.